src/pages/Door.tsx

```diff
@@ -28,7 +28,7 @@ const Door: FC = () => {
   }
 
   return (
-    <Page className="relative">
+    <Page className="relative z-0">
```

Comment:
Tror denne burde ligge fast på `Page` heller, så er issuet fikset for alle pages